packages/main/src/webComponents/Card/Card.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+import { Card } from '@ui5/webcomponents-react/lib/Card';
```

Comment:
Add imports for example:
```suggestion
import '@ui5/webcomponents-icons/dist/icons/person-placeholder.js';
import { Avatar } from '@ui5/webcomponents-react/lib/Avatar';
import { Card } from '@ui5/webcomponents-react/lib/Card';
```